lib/trento/domain/cluster/events/cluster_deregistered.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Trento.Domain.Events.ClusterDeregistered do
+  @moduledoc """
+    This event is emitted when a cluster is decommissioned.
```

Comment:
Why the word `decommissioned`?
We use `Deregisted` all over the code. If you want to have the same approach than in host, you could put in parenthesis